fineract-provider/src/main/java/org/apache/fineract/portfolio/charge/api/ChargesApiResourceSwagger.java

```diff
@@ -64,7 +64,7 @@ static final class GetChargesAppliesToResponse {
             private GetChargesAppliesToResponse() {}
 
             @Schema(example = "1")
-            public Long id;
+            public Integer id;
```

Comment:
This looks wrong to me...It should be Long based on the EnumOptionData line:36

Reply:
Done.